### DISCUSSION TITLE
Is the Public Sponsor Medal permanent?

### DISCUSSION THREAD

**Discussion by Penguin-Eggs** [Schweinepriester/github-profile-achievements#439] [Q&A] [answered]
As the title says.

I sponsored a project and received a medal, but I have now cancelled my subscription sponsorship, will my medal be recycled?

**Comment by Schweinepriester** [comment]
[It will stay](https://github.com/Schweinepriester/github-profile-achievements/discussions/16#discussioncomment-3618680).

(In the past it was removed, but nowadays it stays.)

**Comment by TsaiRongFu** [comment, marked as answer]
As far as I know current sponsorship badges are permanent, if you unsubscribe sponsorships will still show up on your personal page. Even if you use a one-time sponsorship
